sdks/java/maven-archetypes/examples-java8/src/main/resources/archetype-resources/pom.xml

```diff
@@ -50,7 +49,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.5.1</version>
+        <version>@maven-compiler-plugin.version@</version>
```

Comment:
I was thinking it was easier for a reader of the code to understand that the @yyy.version@ is coming from outside without needing to read the properties section. Having the properties section allows for maven archetype users to have a single property location.

Will update and repost.